relayer/strategies.go

```diff
@@ -48,27 +48,35 @@ func relayerMainLoop(ctx context.Context, log *zap.Logger, src, dst *Chain, filt
 		// Apply the channel filter rule (i.e. build allowlist, denylist or relay on all channels available)
 		srcChannels = applyChannelFilterRule(filter, srcChannels)
 
+		// TODO once upstream changes are merged for emitting the channel version in ibc-go,
+		// we will want to add back logic for finishing the channel handshake for interchain accounts.
+		// Essentially the interchain accounts module will initiate the handshake and then the relayer finishes it.
+		// So we will occasionally query recent txs and check the events for `ChannelOpenInit`, at which point
+		// we will attempt to finish opening the channel.
+
 		// Filter for open channels that are not already in our slice of open channels
 		srcOpenChannels = filterOpenChannels(srcChannels, srcOpenChannels)
 
-		// Spin up a goroutine to relay packets & acks for each channel that isn't already being relayed against
-		for _, channel := range srcOpenChannels {
-			if !channel.active {
-				channel.active = true
-				go relayUnrelayedPacketsAndAcks(ctx, log, src, dst, maxTxSize, maxMsgLength, channel, channels)
+		if len(srcOpenChannels) > 0 {
```

Comment:
Since this if-block is the last statement in this loop, inverting it to 

```go
if len(srcOpenChannels) == 0 {
  continue
}
```

would reduce the readers' cognitive overhead a bit here.